scripts/brackets.bat

```diff
@@ -0,0 +1,2 @@
+@echo off
+start Brackets.exe %*
```

Comment:
What does the `start ` prefix do?  In the original implementation, we were just putting INSTALLDIR on the PATH, which meant users were launching Brackets.exe _directly_ when they typed `brackets` on the command line.  So I'm wondering what will be different now.